lib/acpi/acpi_shell.c

```diff
@@ -304,6 +307,8 @@ static int get_acpi_dev_resource(const struct shell *sh, size_t argc, char **arg
 			}
 		}
 
+		irq_res.irq_vector_max = CONFIG_ACPI_IRQ_VECTOR_MAX;
```

Comment:
`ARRAY_SIZE(irqs)` is better, since it's then clear what array this relates to and the code will not break even if you change the size of the array to be derived from some other constant.